test/jdk/java/net/InetAddress/getOriginalHostName.java

```diff
@@ -41,8 +41,8 @@ public class getOriginalHostName {
     public static void main(String[] args) throws Exception {
         final String HOST = "dummyserver.java.net";
         InetAddress ia = null;
-        ia = InetAddress.getByName(HOST);
-        testInetAddress(ia, HOST);
+        ia = getInetAddress(HOST);
+        if (ia != null) testInetAddress(ia, HOST);
```

Comment:
I noticed that's there is one missing test case. This should not throw UnknownHostException either.

```suggestion
        ia = getInetAddress(HOST);
        if (ia != null) testInetAddress(ia, HOST);
        ia = InetAddress.getByAddress(HOST, new byte[] { 1, 2, 3, 4});
        testInetAddress(ia, HOST);
```

Reply:
Thanks. Test `InetAddress.getByAddress("dummyserver.java.net", new byte[] { 1, 2, 3, 4})` has been added and verified locally.